packages/main/src/plugin/openshell-cli/openshell-gateway-manager.spec.ts

```
@@ -0,0 +1,589 @@
+  buildConnectOptions: vi.fn().mockResolvedValue({ gateway: 'http://127.0.0.1:17670' }),
+} as unknown as OpenshellGatewayConfig;
+
+const sdkClientManager = {
+  getClient: vi.fn(),
+} as unknown as OpenshellSdkClientManager;
+
+function encodeVarint(value: number): Buffer {
+  const bytes: number[] = [];
+  while (value > 0x7f) {
+    bytes.push((value & 0x7f) | 0x80);
+    value >>>= 7;
```

Biome (2.5.8)

[error] Assigning a function parameter is confusing.

(lint/style/noParameterAssign)